src/Components/details/index.js

```diff
@@ -1,24 +1,17 @@
 import React, { Component } from 'react';
```

Comment:
having the capitalization problem again, can we make this "Details" vs "details"?